pages/news/2020-XX-XX-concrete-action.html

```diff
@@ -45,7 +45,7 @@
 With your help, we intend to continue the internship process semiannually for at least 5 cycles and at that time assess if we should continue with the program or if our diversification efforts would be better served with a new approach.
 
 An Outreachy intern will cost Mixxx $6,500 to sponsor.
-Because we don't have the money to do this ourselves, we are asking the Mixxx community to step up and pitch in to help raise the necessary funds for this sponsorship.
+As a project that is entirely community-driven, we're relying on you to secure the funding for this effort: There is no company behind Mixxx and all of Mixxx' developers are volunteers that work on Mixxx in their free time.
```

Comment:
As a project that is entirely community-driven, we are looking to you, our collaborators and users, to secure the funding for this effort. There is no company behind Mixxx. All Mixxx developers are volunteers that work on Mixxx in their free time.

Reply:
How necessary is it to reiterate the "There is no company behind Mixxx. All Mixxx developers are volunteers that work on Mixxx in their free time." bit?

Reply:
We often get questions regarding that bit, and I fear without this it might look like we're stingy and not even willing to provide funding for this.

Reply:
I find on Facebook I am constantly reminding users that we have no money, which is why their favorite controller is not supported.  So it feels right to hit that point a little hard

Reply:
adopted the edit, not sure if we want to keep that